desktop/cmp/input/DateInput.js

```diff
@@ -419,6 +419,7 @@ const cmp = hoistCmp.factory(
         }
 
         return div({
+            className: 'xh-date-input__wrapper',
```

Comment:
Thank you for giving this extra wrapper a classname - fixes #2421

Reply:
Ah - two for one! :)